gen/input/src/main/scala/test/StarWarsSubquery.scala

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+// format: off
+/*
+  rules = [GraphQLGen]
+  GraphQLGen.schemaDirs = ["gen/input/src/main/resources/graphql/schemas"]
+ */
+package test
+
+import clue.GraphQLSubquery
+import test.StarWars
+
+@clue.annotation.GraphQLSubquery
+trait StarWarsSubquery extends GraphQLSubquery[StarWars] {
+
+  override val rootType: String = "Character"
```

Comment:
Can we make this a constructor parameter? eg `extends GraphQLSubquery[StarWars]("Character")`

Reply:
Yes, by either changing `GraphQLSubquery` to an `abstract class` or dropping Scala 2 support :) probably these don't need to be `trait`s?

Reply:
I forgot about Scala 2 :D

These don't need to be traits, they can be `class`es.

Reply:
Done :) I also had an idea if we can do `GraphQLSubquery[StarWars, "Character"]` but that will be easier when we are Scala 3 only.